src/test/java/org/opensearch/neuralsearch/common/BaseNeuralSearchIT.java

```diff
@@ -623,6 +624,16 @@ protected void deleteModel(String modelId) {
         );
     }
 
+    @SneakyThrows
```

Comment:
you can remove SneakyThrows from here and add the exception in method signature.

Reply:
Done.